tck/pom.xml

```diff
@@ -48,12 +48,12 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.eclipse.microprofile.apis</groupId>
-            <artifactId>microprofile-config_1.0_api</artifactId>
+            <groupId>org.eclipse.microprofile.config</groupId>
```

Comment:
<groupId>org.eclipse.microprofile.fault.tolerance</groupId>
    <artifactId>microprofile-fault-tolerance-api</artifactId>

Can you change to the above as I meant to depend on fault tolerance api not config api at the moment?